keps/7610-default-local-queue-creation/README.md

```diff
@@ -0,0 +1,216 @@
+# KEP-7610: Create Default LocalQueues Based on NamespaceSelector
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [API Proposal](#api-proposal)
+  - [Controller Logic](#controller-logic)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+  - [Improvements for future versions](#improvements-for-future-versions)
+<!-- /toc -->
+
+## Summary
+
+This KEP proposes a change to the `ClusterQueue` API to introduce an opt-in
+feature that automatically creates a default `LocalQueue` in namespaces that
+match a `ClusterQueue`'s `namespaceSelector`. This avoids the need for administrators
+to manually create a `LocalQueue` in each namespace. 
+
+## Motivation
+
+Kueue `ClusterQueue` can select which namespaces are allowed to submit workloads
+via a `namespaceSelector`. However, this selection only serves as an admission
+rule. For users to actually submit workloads, an administrator must still
+manually create a `LocalQueue` resource in that namespace and point it to the
+appropriate `ClusterQueue`. This manual step is repetitive and could be
+automated.
+
+Automating the creation of a default `LocalQueue` makes the administrator's
```

Comment:
I don't think that cluster configuration management should not be a part of the core Kueue binary and API. While I fully understand that local queue is relatively small and simple, this KEP creates a precedence. Creating Topology, RF, syncing CQs in MulitKueue, etc would also be nice to have featurea but they clearly do not belong to Kueue scope. I would love to see them in one place. What about putting all of the configuration management to a separate binary and APIs?